### ISSUE TITLE
Classification Header

### ISSUE THREAD

**Issue by DuanhaoranCC** [enyac-group/supmae#5]
Thanks for this impressive work and would like to ask the author why the downstream task removes the classification header and then re-adds a new classification header when fine-tuned.

**Comment by Jeff-LiangF** [comment]
Hi @DuanhaoranCC ,

Thanks for your question. For ImageNet-1k fine-tuning, we can reuse the classification head because the # of classes is the same, i.e., 1000. However, our experiments show that random initialization has the same accuracy as the reusing pretraining heads. For other downstream tasks, such as few-shot learning, due to the mismatch of # of classes, we can only initialize a new head. For simplicity, we chose to initialize a new head for all downstream tasks. 

I will close this issue. Feel free to reopen it if you have more questions.
